incubator/kube-registry-proxy/templates/daemon.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     release: {{ .Release.Name | quote }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 spec:
+  selector:
+    matchLabels:
+      release: {{ .Release.Name | quote }}
```

Comment:
@antongorkovenko 

I1016 22:10:27.103] Error: release kube-registry-proxy-u2no0fg1nr failed: DaemonSet.apps "kube-registry-proxy-u2no0fg1nr-kube-registry-proxy" is invalid: spec.template.metadata.labels: Invalid value: map[string]string{"app":"kube-registry-proxy-u2no0fg1nr-kube-registry-proxy"}: `selector` does not match template `labels`

Reply:
it was caused by inconsistency between main and template labels
fixed